src/mono/msbuild/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -129,9 +143,17 @@ public class ApkBuilder
             .Replace("%EntryPointLibName%", Path.GetFileName(entryPointLib));
         File.WriteAllText(Path.Combine(OutputDir, "runtime-android.c"), runtimeAndroidSrc);
         
-        Utils.RunProcess(cmake, workingDir: OutputDir,
-            args: $"-DCMAKE_TOOLCHAIN_FILE={androidToolchain} -DANDROID_ABI=\"{abi}\" -DANDROID_STL=none " + 
-            $"-DANDROID_NATIVE_API_LEVEL={MinApiLevel} -B runtime-android");
+        string cmakeArgs = $"-DCMAKE_TOOLCHAIN_FILE={androidToolchain} -DANDROID_ABI=\"{abi}\" -DANDROID_STL=none " + 
+            $"-DANDROID_NATIVE_API_LEVEL={MinApiLevel} -B runtime-android";
+
+        if (StripDebugSymbols)
+        {
+            // Makefile prints a warrning "clang: warning: argument unused during compilation: '-s'"
+            // but it does use it and calls `android-%abi%-strip` from NDK
+            cmakeArgs+= " -DCMAKE_C_FLAGS=-s";
```

Comment:
maybe we should use `--strip` option of cmake instead: https://cmake.org/cmake/help/latest/manual/cmake.1.html#install-a-project ?

Reply:
Tested and it didn't help so let's keep it as is for now